src/main/java/org/opentripplanner/routing/vehicle_parking/VehicleParkingGroup.java

```diff
@@ -0,0 +1,88 @@
+package org.opentripplanner.routing.vehicle_parking;
+
+import java.util.Locale;
+import java.util.Objects;
+import javax.annotation.Nullable;
+import org.opentripplanner.transit.model.basic.I18NString;
+import org.opentripplanner.transit.model.framework.FeedScopedId;
+
+/**
+ * Group of vehicle parking locations.
+ * <p>
+ * All fields are immutable. If any other properties change a new VehicleParkingGroup instance
+ * should be created.
+ */
+public class VehicleParkingGroup {
+
+  private final FeedScopedId id;
+
+  private final I18NString name;
+
+  private final double x, y;
+
+  VehicleParkingGroup(VehicleParkingGroupBuilder vehicleParkingGroupBuilder) {
+    this.id = vehicleParkingGroupBuilder.id;
+    this.name = vehicleParkingGroupBuilder.name;
+    this.x = vehicleParkingGroupBuilder.x;
+    this.y = vehicleParkingGroupBuilder.y;
+  }
+
+  public static VehicleParkingGroupBuilder builder() {
+    return new VehicleParkingGroupBuilder();
+  }
+
+  /**
+   * The id of this vehicle parking group, prefixed by the source(=feedId) so that it is unique.
+   */
+  public FeedScopedId id() {
+    return id;
+  }
+
+  /**
+   * The name of this vehicle parking group, which may be translated when displaying to the user.
+   */
+  @Nullable
+  public I18NString name() {
+    return name;
+  }
+
+  /**
+   * Longitude
+   */
+  public double x() {
```

Comment:
I understand that this is the same in `VehicleParking` but I always found the name x/y confusing even though they are the correct mathematical terms. What do you think about renaming it latitude/longitude? If you want to be extra-nice you can rename it in `VehicleParking` too.

Reply:
Actually, should the coordinates be replaced with a `WgsCoordinate` instance?

Reply:
Good idea!

Reply:
Updated in https://github.com/opentripplanner/OpenTripPlanner/pull/4510/commits/d5ead8bdad8b0597d8a9e12cbfab66189a9d9df6 and https://github.com/opentripplanner/OpenTripPlanner/pull/4510/commits/65086c27329359c5e6940e21c651c3b4d198a017